bp_be/src/include/bp_be_vm_defines.vh

```diff
@@ -4,31 +4,31 @@
 
 `define declare_bp_sv39_pte_s(pte_width_mp, ppn_width_mp) \
   typedef struct packed {                                 \
+    logic [pte_width_mp-10-ppn_width_mp-1:0] reserved;    \
```

Comment:
I believe this is defined in bp_common fe_be_if already

Reply:
This is the pte as is saved in the memory. I didn't find the definition in common files. Can you point to the location if you find it?

Reply:
Oh yeah, you're right